pvlib/atmosphere.py

```diff
@@ -406,20 +407,58 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
     .. [1] Gueymard, Christian. SMARTS2: a simple model of the atmospheric
        radiative transfer of sunshine: algorithms and performance
        assessment. Cocoa, FL: Florida Solar Energy Center, 1995.
-
-    .. [2] Marion, William F., et al. User's Manual for Data for Validating
+    .. [2] Lee, Mitchell, and Panchula, Alex. "Spectral Correction for
+       Photovoltaic Module Performance Based on Air Mass and Precipitable 
+       Water." IEEE Photovoltaic Specialists Conference, Portland, 2016 
+    .. [3] Marion, William F., et al. User's Manual for Data for Validating
        Models for PV Module Performance. National Renewable Energy
        Laboratory, 2014. http://www.nrel.gov/docs/fy14osti/61610.pdf
     """
 
+    import pdb
```

Comment:
You'll need to delete these pdb lines. They prevent the test suite from completing.